test/runtime/connectivity.go

```diff
@@ -21,7 +35,6 @@ func runOnNetNextOnly(f func()) func() {
 	return func() {}
 }
 
-var _ = Describe("RuntimeConnectivityInVethModeTest", runtimeConnectivityTest("veth"))
```

Comment:
I don't fully understand this commit message - doesn't d391e52ee93777a031ec9337aa65b8d01eb2c63d add the way of plumbing the labels through to `cilium-agent`?

Reply:
Yes it does but it only works for containers with the network managed by libnetwork, not CNI.